README.md

```diff
@@ -72,7 +72,7 @@ Open the update server in your browser:
 
 [localhost:8080](http://localhost:8080)
 
-See below for how to build and run the update server on your own. In addition we have a [guide](http://www.eclipse.org/hawkbit/documentation/guide/runhawkbit.html) for setting up a complete landscape.
+See below for how to build and run the update server on your own. In addition we have a [guide](https://www.eclipse.org/hawkbit/gettingstarted/) for setting up a complete landscape.
```

Comment:
Would you mind, changing the link to `https://www.eclipse.org/hawkbit/guides/runhawkbit/`?

Reply:
Thanks for pointing this out @laverman , I'll update this.